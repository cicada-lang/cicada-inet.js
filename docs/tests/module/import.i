import zero, one, add from "../datatypes/Nat.i"

begin zero one add inspect run inspect end
