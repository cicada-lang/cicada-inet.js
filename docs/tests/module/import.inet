import zero, one, add from "../datatypes/Nat.i"