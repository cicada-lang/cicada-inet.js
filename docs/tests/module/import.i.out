(zero₁₅)-value addend-(add₆)
(add1₁₇)-value target-(add₆)
(zero₁₆)-value prev-(add1₁₇)
(add₆)-return

(zero₁₅)-value addend-(add₆)
(add1₁₇)-value target-(add₆)
(zero₁₆)-value prev-(add1₁₇)
(add₆)-return

